components/renku_data_services/migrations/versions/a06e2c511bea_run_missed_migrations.py

```diff
@@ -0,0 +1,69 @@
+"""run missed migrations
+
+Revision ID: a06e2c511bea
+Revises: e16782d6bf32
+Create Date: 2024-03-11 16:23:52.894901
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "a06e2c511bea"
+down_revision = "e16782d6bf32"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
+        "projects", "creation_date", existing_type=postgresql.TIMESTAMP(timezone=True), nullable=True, schema="projects"
+    )
```

Comment:
I think it would be nice to have this column not be nullable. Maybe the orm file needs to specify `nullable=False`?

Reply:
Fixed.